### PR TITLE
Django 1.11 tests compatibility

### PR DESCRIPTION
**Pull request by saifrim**
Fix for unit tests to be runned on:
Python 3.4.0
Django 1.11.16

The modifications are necessary to prove that tests work on the new environment.

### CODE REVIEW

File: impersonate/templates/impersonate/list_users.html [hunk 1]
```diff
@@ -1,4 +1,3 @@
-{% load url from future %}
```
Comment (saifrim) [inline impersonate/templates/impersonate/list_users.html]: https://docs.djangoproject.com/en/2.1/releases/1.5/
>  If you were using {% load url from future %} you can simply remove that line under Django 1.5

File: impersonate/templates/impersonate/search_users.html [hunk 2]
```diff
@@ -1,4 +1,3 @@
-{% load url from future %}
```
Comment (saifrim) [inline impersonate/templates/impersonate/search_users.html]: https://docs.djangoproject.com/en/2.1/releases/1.5/
>  If you were using {% load url from future %} you can simply remove that line under Django 1.5

File: runtests.py [hunk 3]
```diff
@@ -27,6 +28,23 @@
         'django.contrib.admin',
         APP_NAME,
     ),
+    TEMPLATES = [
```
Comment (saifrim) [inline runtests.py]: https://docs.djangoproject.com/en/2.1/releases/1.8/#template-related-settings

**Comment by asuiu** [comment]
@saifrim is it something up to date, or we can close it?

**Pull request closed without merging**